Movies Time/ViewController/MovieDetailsViewController.swift

```diff
@@ -197,14 +197,14 @@ class MoviesDetailsViewController: UIViewController {
         overviewText.text = movie.overview
     }
     
-    private func viewsFor() -> UIView{
+    private func viewsProperties() -> UIView{
```

Comment:
this still doesn't look quite good, I should named it like "createGenericView" or "createReusableView", also there must always be a space before the open bracket "{" in all methods.